README.md

```diff
@@ -34,6 +61,21 @@ and configuring the emulator manually.
 
 See [firebase-emulator](./firebase-emulator) folder for additional details.
 
+Or start the emulator with the following command:
+
+```bash
+docker run \
+  --rm \
+  -p=9000:9000 \
```

Comment:
Not sure why we publish this many ports. Please review. Ideally, I would comment on the purpose of these.

Reply:
Firebase emulators suite gives us 6 emulators and we're exposing them all in this example. The emulator's readme provides more notes on this point.
I'll add a note that we're starting all default emulators in this example.